cmd/groups.go

```diff
@@ -34,25 +34,66 @@ var addGroupCmd = &cobra.Command{
 	Use:     "group",
 	Aliases: []string{"g"},
 	Short:   "Add a group to lagoon",
```

Comment:
This command is now way more complex and the short description is insufficient to cover the additions made.
I would like to see the short description updated, and an additional `Long` description added.

Something like `Add a group to lagoon, or add a group to an organization`
With the long text including additional help information like 
```
To add a group to an organization, you'll need to include the `organization` flag and provide the name of the organization. You need to be an owner of this organization to do this.
If you're the organization owner and want to grant yourself ownership to this group to be able to deploy projects that may be added to it, specify the `owner` flag, otherwise you will still be able to add and remove users without being an owner
```

Just some spicy thoughts though, `addOrgOwner` is only used by the project creation resolver so that the person creating a project in an organization would be added to the project group when it is created. This is because the default behaviour for organization owners is that they can manage the basic elements of an organization, but not be able to trigger deployments etc. Sort of like a walled garden to prevent unnecessary permissions. Unfortunately this an lead to situations where some users may need to be added to a group. So this flag is there mostly as a way to prevent a two step process which would be like this normally, and is the same process a user would have to do via the UI.

* create group `add group`
* add self to group with appropriate role `add user-group`

Instead, this flag gives a gateway directly to `owner` role within that group. 

I could be convinced to leave it here if the flag was changed from `org-owner` (as this is a bit confusing) and the wording around it was adjusted based on the long description text

Reply:
Yeah I was torn whether to include the `org-owner` flag at all, happy to nuke it. In the interim I've updated the flag to `owner` and expanded the usage description, let me know what you think.